pkg/ast/spl/spl.peg

```diff
@@ -1853,6 +1855,11 @@ StatisticBlock <- PIPE statisticExpr:StatisticExpr {
         GroupByColumns: groupByColumns,
     }
 
+	if config.IsNewQueryPipelineEnabled() {
+		root.GroupByRequest = aggNode.GroupByRequest
+		return root, nil
+	}
+
     return aggNode, nil
```

Comment:
The way we process top/rare command is we create two Aggregators chained together. 
The first Agg contains the Group By Request and the second agg contains the actual Top/Rare expr.

In the new pipeline I want them to be in the same Aggregator not separate, so I had to return `root` instead of aggNode.